tailwind.config.js

```diff
@@ -109,6 +108,26 @@ module.exports = {
       spacing: {
         '3xs': '.125rem',
         xxs: '.25rem',
+        xxs1: '.375rem',
+        xs: '.5rem',
+        xs1: '.563rem',
+        s: '.75rem',
+        s1: '.76rem', //12.2px
+        s2: '.813rem', //13px
+        s3: '.833rem', //13.33px
+        m: '1rem',
+        m1: '1.125rem',
+        l: '1.5rem',
+        xl: '2rem',
+        xxl: '2.5rem',
+        xxl1: '2.875rem',
+        '3xl': '3rem',
+        '3xl1': '3.625',
+        '4xl': '4rem',
+        '5xl': '8rem'
+      },
+      backgroundOpacity: {
+        24: '0.24',
```

Comment:
Please revise the need for all of these values.
For example, the lengths of input fields don't have to be exactly as the design of atom, the lengths might change when put in context, molecules and organisms.
Also, some of these already exist in tailwind, like `8rem` and `2rem`. We are only extending here so they are still there.